config/qt5_core.mpb

```diff
@@ -0,0 +1,56 @@
+project {
+  expand(QT5_INCDIR) {
+    $QT5_INCDIR
+    $(QT5_DIR)/include
+  }
+
+  requires += qt5
+  includes += $(QT5_INCDIR)
+  libpaths += $(QT5_DIR)/lib
+  libs += Qt5Core
+
+  // On Linux at least, Qt5 by default requires position independent code
+  verbatim (make, macros) {
+    CPPFLAGS += -fPIC
```

Comment:
I tested with `-type make` on Linux.  It uses -fPIC by default even when building executables.